docs/core/resilience/index.md

```diff
@@ -0,0 +1,152 @@
+
+Consider the following string-based `key` example:
+
+```csharp
+using Microsoft.Extensions.DependencyInjection;
+using Polly;
+using Polly.CircuitBreaker;
+using Polly.Registry;
+using Polly.Retry;
+
+var services = new ServiceCollection();
+
+const string key = "Retry-CircuitBreaker-Timeout";
+
+services.AddResiliencePipeline(key, builder =>
+{
+    builder.AddRetry(new RetryStrategyOptions());
+
+    builder.AddCircuitBreaker(new CircuitBreakerStrategyOptions());
+
+    builder.AddTimeout(TimeSpan.FromSeconds(5));
+
+    // Add other strategies here...
+});
+```
+
+The preceding code:
+
+- Creates a new `ServiceCollection` instance.
+- Defines a `key` to identify the pipeline.
+- Adds a resilience pipeline to the `ServiceCollection` instance.
+- Configures the pipeline with a retry strategy, circuit breaker strategy, and timeout strategy.
+
+Each pipeline is configured for a given `key`, and each `key` is used to identify it's corresponding `ResiliencePipeline` when getting the pipeline from the provider. The `key` is a generic type parameter of the `AddResiliencePipeline` method.
+
+### Resilience pipeline builder extensions
+
+To add a strategy to the pipeline, call any of the available `Add*` extension methods on the `ResiliencePipelineBuilder` instance.
+
+- `AddRetry`: Try again if something fails, which is useful when the problem is temporary and might go away.
+- `AddCircuitBreaker`: Stop trying if something is broken or busy, which benefits you by avoiding wasting time and making things worse.
+- `AddTimeout`: Give up if something takes too long, which can improve performance by freeing up resources.
+- `AddRateLimiter`: Limit how many requests you make or accept, which enables you to control load.
+- `AddFallback`: Do something else when experiencing failures, which improves user experience.
+- `AddHedging`: Do more than one thing at the same time and take the fastest one, which can improve responsiveness.
+
+For more information, see [Resilience strategies](https://www.pollydocs.org/strategies/index.html).
+
+## Enrichment
+
+Enrichment is the automatic augmentation of telemetry with well-known state, in the form of name/value pairs. For example, an app might emit a log that includes the _operation_ and _result code_ as columns to represent the outcome of some operation. In this situation and depending on peripheral context, enrichment adds _cluster name_, _process name_, _region_, _tenant id_ and more to the log as it's sent to the telemetry backend. Enrichment is done systematically and behind the scenes relative to logging and metrics update within the app's main code.
```

Comment:
This part of the sentence doesn't make sense to me. Should it be _"updated"_?

> relative to logging and metrics update within the app's main code

Reply:
Thank you @martincostello - but I'm not sure what part of this paragraph you're referring to. Could you please be a bit more specific?